.github/workflows/build.yml

```diff
@@ -56,7 +56,7 @@ jobs:
         run: |
           cd main
           python -m pip install --upgrade pip
-          pip install git+https://github.com/PennyLaneAI/pennylane.git
+          pip install pennylane --upgrade
```

Comment:
This is redundant and would need to be removed such that we pull PL from `requirements.txt` (https://github.com/PennyLaneAI/pennylane-lightning/pull/43#discussion_r505728317)
```suggestion
```